snuba/clickhouse/astquery.py

```diff
@@ -91,7 +92,14 @@ def _sql_data_list(self) -> Sequence[Tuple[str, str]]:
 
         where_clause = ""
         if self.__condition:
-            formatted_condition = self.__condition.accept(formatter)
+            if state.get_config("infix_where_format", 0):
```

Comment:
What's the reason for using config to control this value? (as opposed to settings). Is it supposed to be a permanent option we are providing the operator of Snuba with or something temporary for the roll out?

Reply:
Temporary rollout just to be sure we can shutdown quickly if something is wrong. If this breaks, no query works potentially.

Reply:
Not sure how long we plan to leave it for, would be great if you can add a small comment inline to note this. Hard to keep remember what's meant to be temporary vs permanent in config sometimes.